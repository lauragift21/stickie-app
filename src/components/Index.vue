<template>
  <div class="container h-screen mt-6 mx-4">
    <board />
    <guide />
  </div>
</template>

<script>
import Board from './Board'
import Guide from './Guide'

export default {
  components: {
    Board,
    Guide,
  },
}
</script>
