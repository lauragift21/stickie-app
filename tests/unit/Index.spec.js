import { render, screen } from '@testing-library/vue'
import Index from '../../src/components/Index'
jest.mock('../../src/components/Index')

describe('Index', () => {
  it('should render Index page', () => {
    render(Index)
    expect(Index).toMatchSnapshot()
  })
})
