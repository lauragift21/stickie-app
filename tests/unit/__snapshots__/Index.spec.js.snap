// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render Index page 1`] = `
Object {
  "$_vueTestUtils_original": [Circular],
  "_Ctor": Object {},
  "components": Object {
    "Board": Object {
      "data": [MockFunction],
      "methods": Object {
        "addSticky": [MockFunction],
        "deleteSticky": [MockFunction],
        "detectClick": [MockFunction],
        "editSticky": [MockFunction],
        "handleDragEnd": [MockFunction],
      },
      "mounted": [MockFunction],
      "render": [MockFunction],
      "staticRenderFns": Array [],
    },
    "Guide": Object {
      "render": [MockFunction],
      "staticRenderFns": Array [],
    },
  },
  "render": [MockFunction] {
    "calls": Array [
      Array [
        [Function],
      ],
    ],
    "results": Array [
      Object {
        "type": "return",
        "value": undefined,
      },
    ],
  },
  "staticRenderFns": Array [],
}
`;
